lib/semian/rails.rb

```diff
@@ -0,0 +1,7 @@
+require 'active_record/connection_adapters/abstract_adapter'
+
+class ActiveRecord::ConnectionAdapters::AbstractAdapter
```

Comment:
Won't that make it order dependant such that it won't work if semiannual is loaded first by bundler